app/controllers/pages_controller.rb

```diff
@@ -2,7 +2,11 @@ class PagesController < ApplicationController
   skip_before_action :authenticate_user!
 
   def home
-    @referral = Referral.new
+    if user_signed_in?
```

Comment:
let's review this one tomorrow as well, i'm guessing this is related to the e-mail invite on home page. I've not seen the commits on that, so probably best to review together.